src/org/parosproxy/paros/view/MainMenuBar.java

```diff
@@ -104,15 +106,16 @@ public MainMenuBar() {
 	 * This method initializes this
 	 */
 	private void initialize() {
-        this.add(getMenuFile());

```

Comment:
This should be reverted.

Reply:
Do you mean `this.add(getMenuImport())` should be removed?

Reply:
No he means you shouldn’t have removed the addition of the file menu 😀

Reply:
Yeah, I know, but the point is about not doing these kind of changes in the first place. Not a blocker though.